<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vizualizar_window</class>
 <widget class="QMainWindow" name="vizualizar_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>237</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(44, 44, 44);
color: rgb(255, 105, 182);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-20</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Password Manager</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>231</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>248</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMENU">
    <property name="title">
     <string>MENU</string>
    </property>
    <addaction name="actionVisualizar_Senhas"/>
    <addaction name="actionGerar_senhas"/>
    <addaction name="actionAdicionar_senhas"/>
   </widget>
   <addaction name="menuMENU"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionVisualizar_Senhas">
   <property name="text">
    <string>Visualizar senhas</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionGerar_senhas">
   <property name="text">
    <string>Gerador de senhas</string>
   </property>
  </action>
  <action name="actionAdicionar_senhas">
   <property name="text">
    <string>Adicionar senhas</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
